internals/config/default.yml

```diff
@@ -0,0 +1,244 @@
+        email:
+          type: "email"
+          pattern: '.+@.+\..+'
+      contact_text: true
+      login_page_link: true
+
+    password_reset_confirm_form:
+      additional_text: true
+      input_fields:
+        password:
+          type: "password"
+          pattern: ".{6,}"
+        password_confirm:
+          type: "password"
+          pattern: ".{6,}"
+      contact_text: true
+      login_page_link: true
+
+    contact_page:
+      email: "support@openwisp.co"
+      helpdesk: "+789 948 564"
+      social_links:
+        - alt:
+            en: "twitter"
+          icon: "twitter.svg"
+          url: "https://twitter.com/openwisp"
+        - alt:
+            en: "facebook"
+          icon: "facebook.svg"
+          url: "https://facebook.com/openwisp"
+
+    status_page:
+      links:
+        - text:
+            en: "Change your password"
+          url: "/{orgSlug}/change-password/"
+
+    login_form:
+      social_login:
+        links:
+          - text:
+              en: "Login with Facebook"
+            url: "#facebook"
+            icon: "facebook.svg"
+          - text:
+              en: "Login with Google"
+            icon: "google.svg"
+            url: "#google"
+          - text:
+              en: "Login with Twitter"
+            icon: "twitter.svg"
+            url: "#twitter"
+      input_fields:
+        username:
+          type: "email"
+          pattern: '.+@.+\..+'
+        password:
+          type: "password"
+          pattern: ".{6,}"
+        remember_me:
+          type: "checkbox"
+          value: true
+      additional_info_text: true
+      buttons:
+        register: true
+      links:
+        forget_password: true
+
+    password_change_form:
+      input_fields:
+        password1:
+          type: "password"
+          pattern: ".{6,}"
+        password2:
+          type: "password"
+          pattern: ".{6,}"
+
+    404_page:
+      heading: true
+      sub_heading: true
+      message: true
+      homepage_link: true
+
+    captive_portal_login_form:
+      method: post
+      action: http://localhost:8000/captive-portal-mock/login/
+      fields:
+        username: auth_user
+        password: auth_pass
+      macaddr_param_name: macaddr
+      additional_fields:
+        - name: zone
+          value: zone_name
+        - name: redirurl
+          value: http://localhost:8080/default/status
+        - name: accept
+          value: accept
+
+    captive_portal_logout_form:
+      method: post
+      action: http://localhost:8000/captive-portal-mock/logout/
+      fields:
+        id: logout_id
+      additional_fields: []
+      logout_by_session: true
+    
+    mobile_phone_verification_form:
+      input_fields:
+        code:
+          type: "text"
+          pattern: '[0-9]{6,}'
+
+    phone_number_change_form:
+      buttons:
+        change_phone_number: true
+        cancel: true
+
+  privacy_policy:
+    en: "privacy-en.md"
+  terms_and_conditions:
+    en: "terms-en.md"
```

Comment:
always on the concern of size, we could change these files to have only a few words in them because they're not going to be used anyway.

Reply:
Should I change the content of these files? 
Also, the whole content is present in the configuration, Can we use express to serve these files' content?

Reply:
> Should I change the content of these files?
Yes.

> Also, the whole content is present in the configuration, Can we use express to serve these files' content?

Definitely! That's a great idea! :+1: